### ISSUE TITLE
Create ZSH Script for switching between organizations

### ISSUE THREAD

**Issue by paul-thompson1** [PublicDataWorks/complaint-manager#585]
**Context:** I am creating a bash script to automate the process of switching between organizations on a developer’s local development environment. So that developer’s can save time while developing.

The script will do the following:
- Prompts the user for environment variables if none are detected in .zshrc.
- Changes the environment variable to the other organization in .zshrc.
- Changes ORG variable in docker-compose.yml

**Note:** The script will be in the scripts directory in the project repository once complete

**Comment by paul-thompson1** [comment]
I finished the bash script and tested it on my local machine. I just need to make sure the script works on others machines.
**Here is how the script looks when the user first runs it**
![INITIAL_SETUP](https://github.com/PublicDataWorks/complaint-manager/assets/159827724/53ca05ea-1728-47bc-9a11-f5267912a4fb)

**Here is how the script looks when the user has run the script a second time**
![SECOND_PASS](https://github.com/PublicDataWorks/complaint-manager/assets/159827724/ddf9ece0-17ab-42c8-b54a-1de9c6ada7ee)

**Comment by paul-thompson1** [comment]
I requested and received feedback on the script from multiple people.

I added the following:
- Ensure the provided directory exists and includes the instance-files subdirectory.
- The script can now be run from the root directory of complaint_manager.
- Provide more meaningful messages when using the script.

Here is the link to the script with updated changes: [link](https://github.com/portalman1994/change_organization)
